kitsune/wiki/jinja2/wiki/includes/document_macros.html

```diff
@@ -427,9 +427,9 @@ <h2 class="sumo-page-subheading">{{ _('In English') }}</h2>
           <div class="topic-article--text">
             <h2 class="sumo-card-heading">
               <a {% if is_fallback %} href="{{ document['url']|replace('/'+ settings.WIKI_DEFAULT_LANGUAGE +'/', '/' + request.LANGUAGE_CODE + '/') }}" {% else %} href="{{ document['url'] }}" {% endif %}
-                data-event-category="device-migration-tabbed-menu" 
+                data-event-category="device-migration-tabbed-menu"
                 data-event-action="link-click"
-                data-event-label="{{ active_topic_slug }}">
+                data-event-label="{{ document['document_title'] }}">
```

Comment:
In the initial implementation of the tabbed menu, we were using the `subtopic.title` but we swapped that for the `slug` b/c of the [concern](https://github.com/mozilla/kitsune/pull/5475#discussion_r1192606494) that introducing spaces can be prone to errors. I agree that title is better especially for GA, I just want to make sure that we don't foresee any problems with the spaces.

Reply:
@akatsoulas I had the same concern, thinking that perhaps we should use the document slug instead. It looks like we've used the document title for other links in the past for GA, but I still wonder about it.